docs/vsock.md

```diff
@@ -56,8 +56,8 @@ Client A initiates connection to Server A in [figure below](#vsock-connections):
 1. Guest: `accept()` the new connection.
 1. Host: `read()` "OK `<assigned_hostside_port>`\n".
 
-The channel is established between the sockets obtained at steps 3 (host)
-and 5 (guest).
+The channel is established between the sockets obtained at steps 4 (host)
```

Comment:
Step 3 (assuming ```socket()``` was called before ```connect()```) and step 5 (```accept()```) look like ways to obtain sockets to me. Why is it wrong? Do you suggest that ```send()``` and ```read()``` obtain sockets instead?

Reply:
... I could have sworn it said "port numbers" instead of sockets. You're right, my change is nonsense